src/components/InnerNavigation.js

```diff
@@ -0,0 +1,46 @@
+import styled from 'styled-components';
+import PAGES from '../assets/pages';
+
+export default function InnerNavigation({ currentPage, handleNavigation }) {
+  return (
+    <Wrapper>
+      <StyledDetailsButton currentPage={currentPage} onClick={handleNavigation}>
+        Details
+      </StyledDetailsButton>
+      <StyledSeasonButton currentPage={currentPage} onClick={handleNavigation}>
```

Comment:
I'd pass the page details to the `handleNavigation` function:
```suggestion
      <StyledDetailsButton currentPage={currentPage} onClick={() => handleNavigation('deatils')}>
        Details
      </StyledDetailsButton>
      <StyledSeasonButton currentPage={currentPage} onClick={() => handleNavigation('seasons')}>
```